examples/sum/.vscode/settings.json

```diff
@@ -0,0 +1,25 @@
+{
```

Comment:
This is a bit of a heavy config file in an example, could we remove it?

Reply:
Sure, it was added by `cargo prove new`, fyi.